eodag/resources/providers.yml

```diff
@@ -1819,10 +1821,12 @@
         tileIdentifier:
           - '{{"query":{{"mgrs:utm_zone":{{"eq":"{tileIdentifier#slice_str(0,2,1)}"}},"mgrs:latitude_band":{{"eq":"{tileIdentifier#slice_str(2,3,1)}"}},"mgrs:grid_square":{{"eq":"{tileIdentifier#slice_str(3,5,1)}"}}}}}}'
           - '{utmZone}{latitudeBand}{gridSquare}'
-    S2_MSI_L2A:
+    S2_MSI_L2A_JP2:
       productType: sentinel-2-l2a
       metadata_mapping:
-        id: '{$.properties."s2:product_uri"#remove_extension}'
+        id:
```

Comment:
this doesn't work, please see how to configure query-string just bellow (`utmZone` for instance) and test if you can search by id using the updated configuration